app/uk/gov/hmrc/apiplatformoutboundsoap/controllers/ConfirmationController.scala

```diff
@@ -42,7 +42,7 @@ class ConfirmationController @Inject()(cc: ControllerComponents,
     def callService(deliveryStatus: DeliveryStatus, id: String): Future[Result] = {
       if(id.trim.nonEmpty) {
         confirmationService.processConfirmation(xml, id.trim, deliveryStatus) map {
-          case NoContentUpdateResult => NoContent
+          case NoContentUpdateResult => Accepted
```

Comment:
Should we rename this case object name to reflect the http status, something like AcceptedUpdateResult ??